ublox-short-range/src/error.rs

```diff
@@ -56,6 +56,10 @@ pub enum WifiConnectionError {
     WaitingForWifiDeactivation,
     BufferOverflow,
     // SsidNotFound,
+    /// Config id above 9.
+    IncompatiableConfigId,
```

Comment:
This error kind seems unused? Is it intended to be used later, or is there something missing in this PR?

Reply:
It is for later use